modules/sdk-unified-wallet/src/ecdsa-unified-wallet/types.ts

```diff
@@ -1,4 +1,5 @@
-import { CoinFamily } from '@bitgo/statics';
+import { Gteth, Eth } from '@bitgo/sdk-coin-eth';
+import { Tpolygon, Polygon } from '@bitgo/sdk-coin-polygon';
 
-export const supportedCoins: string[] = [CoinFamily.ETH, CoinFamily.POLYGON];
-export type EvmWalletCoins = typeof supportedCoins[number];
```

Comment:
Seems to be breaking change?, Can you kindly mention in the commit message ([format](https://www.conventionalcommits.org/en/v1.0.0/#:~:text=in%20Semantic%20Versioning\).-,BREAKING,-CHANGE%3A%20a))